### ISSUE TITLE
Incorrect export of scenes

### ISSUE THREAD

**Issue by Sinled** [farling42/fvtt-export-markdown#10]
Hi, I am trying to export scenes from "Tales from the Shadows" and all coordinates are messed up in leaflet block, they have `NaN` instead of numbers

here are example of export

````markdown
---
title: "01. Brightsigh Tower Environs"
icon: ":map:"
aliases: "01. Brightsigh Tower Environs"
foundryId: Scene.FDsBb3U3zgCrCOE0
tags:
  - Scene
---

# 01. Brightsigh Tower Environs

```leaflet
id: Scene.FDsBb3U3zgCrCOE0
bounds:
    - [0, 0]
    - [450, 330]
defaultZoom: 2
lat: 225
long: 165
height: 100%
draw: false
unit: ft
showAllMarkers: true
preserveAspect: true
image: [[modules-kp-tales-from-the-shadows-assets-maps-Map-05A-Brightsigh-Tower-Environs.webp]]
imageOverlay:
    - [ [[modules-kp-tales-from-the-shadows-assets-maps-Map-05A-Brightsigh-Tower-Environs-Keys.webp|Map-05A-Brightsigh-Tower-Environs-Keys.webp]], [NaN, NaN], [NaN, NaN] ]
    - [ [[modules-kp-tales-from-the-shadows-assets-maps-Map-05A-Brightsigh-Tower-Environs-Title.webp|Map-05A-Brightsigh-Tower-Environs-Title.webp]], [NaN, NaN], [NaN, NaN] ]
marker: default, NaN, NaN, [[1. A Tower of Jagged Glass]]
marker: default, NaN, NaN, [[2. Darakhul Encampment]]
marker: default, NaN, NaN, [[3. Haunt Site]]
marker: default, NaN, NaN, [[4. Valstitz Winery]]
marker: default, NaN, NaN, [[5. Lethe-Tainted Well]]
marker: default, NaN, NaN, [[6-11. Residences]]
marker: default, NaN, NaN, [[6-11. Residences]]
marker: default, NaN, NaN, [[6-11. Residences]]
marker: default, NaN, NaN, [[6-11. Residences]]
marker: default, NaN, NaN, [[6-11. Residences]]
marker: default, NaN, NaN, [[6-11. Residences]]
marker: default, NaN, NaN, [[12. Well]]
marker: default, NaN, NaN, [[13. Iann's House]]
marker: default, NaN, NaN, [[14. Shrine to Sarastra]]
marker: default, NaN, NaN, [[16. Fairy Ring]]
marker: default, NaN, NaN, [[15. Fiona's House]]
````

and here this scene json from v12 foundry
[fvtt-Scene-01.-brightsigh-tower-environs-FDsBb3U3zgCrCOE0.json](https://github.com/user-attachments/files/17290591/fvtt-Scene-01.-brightsigh-tower-environs-FDsBb3U3zgCrCOE0.json)

**Comment by farling42** [comment]
Fixed in 0.12.2

**Comment by Sinled** [comment]
Thank you for quick fix!
